source/includes/_rf.md

```diff
@@ -146,7 +146,7 @@ postcode | <span class="label label-required">Required</span> A valid postcode o
 country | <span class="label label-required">Required</span> USA only.
 schedule | D (Daily), W (Weekly) or M (Monthly). You to purchase an add on before you can use daily reporting. Defaults to M (Monthly).
 run-on | Numeric day of week or day of month to run the report on (applicable to weekly and monthly schedules). Defaults to current day of month. If you create your report today it'll be run on the 17th of each month unless you specify otherwise.
-receive-email-alerts | One of 1 or 0. If set to 1 we will send report alerts to all email addresses specified (see field below). If you include customer email addresses when setting up your report we'll also email them the alerts so please be sure this is what you want before adding their addresses. Defaults to 0.
+receive-email-alerts | One of 1, 2 or 0. If set to 1 we will send report alerts every time report finishes (no matter new reviews were found or not) to all email addresses specified (see field below). If you include customer email addresses when setting up your report we’ll also email them the alerts so please be sure this is what you want before adding their addresses. If set to 2 we will only send email alerts if new reviews were found upon report completion. Defaults to 0.
```

Comment:
One of 0, 1 or 2. If set to 1 we will send report alerts every time a report finishes (regardless of whether or not new reviews were found) to all email addresses specified (see field below). If you include customer email addresses when setting up your report we’ll also email them the alerts so please be sure this is what you want before adding their addresses. If set to 2 we will only send email alerts if new reviews were found upon report completion. Defaults to 0 which means no alerts will be sent.